src/containers/AddCard/AddCard.jsx

```diff
@@ -26,10 +26,12 @@ class AddCard extends Component {
 
   componentWillMount() {
     document.addEventListener('mousedown', this.handleClick, false);
+    document.querySelector('.columnContainer').classList.add('blur');
   };
 
   componentWillUnmount() {
     document.removeEventListener('mousedown', this.handleClick, false);
+    document.querySelector('.columnContainer').classList.remove('blur');
```

Comment:
commented
Instead of using document, try to directly add the class in the render function using className and the ternary operator